pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.web3j.sample</groupId>
+    <artifactId>sample-project-maven</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
I would encourage semantic versioning - e.g. 1.0.0-SNAPSHOT providing Maven plays nicely with it.